docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,199 +1,339 @@
+  "normal test".
 
-[CHILD_FOUND]: #outcomes
 
-[NO_CHILD]: #outcomes
+## Summary
+
+Message Tag                | Level | Arguments                             | Message ID for message tag
+:--------------------------|:------|:--------------------------------------|:--------------------------
+B01_CHILD_IS_ALIAS         |NOTICE |domain_child, domain_target, ns_ip_list| "{domain_child}" is not a zone. It is an alias for "{domain_target}". Run a test for "{domain_target}" instead. Returned from name servers "{ns_ip_list}.
+B01_CHILD_FOUND            |INFO   | domain                                | The zone "{domain}" is found.
+B01_CHILD_NOT_EXIST        |INFO   | domain                                | "{domain}" does not exist as it is not delegated.
+B01_INCONSISTENT_ALIAS     |ERROR  | domain                                | The alias for "{domain}" is inconsistent between name servers.
+B01_INCONSISTENT_DELEGATION|ERROR  |domain_child, domain_parent, ns_ip_list| The name servers for parent zone "{domain_parent}" give inconsistent delegation of "{domain_child}". Returned from name serververs "{ns_ip_list}".
+B01_NO_CHILD               |ERROR  | domain_child, domain_super            | "{domain_child}" does not exist as a DNS zone. Try to test "{domain_super}" instead.
+B01_PARENT_FOUND           |INFO   | domain, ns_ip_list                    | The parent zone is "{domain}" as returned from name servers "{ns_ip_list}".
+B01_PARENT_INDETERMINED    |WARNING| ns_ip_list                            | The parent zone cannot be determined on name servers "{ns_ip_list}".
+B01_UNEXPECTED_NS_RESPONSE |WARNING|domain_child, domain_parent, ns_ip_list| Name servers for parent domain "{domain_parent}" give an incorrect response on SOA query for "{domain_child}". Returned from name servers {ns_ip_list}.
+
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+
+## Test procedure
+
+In this section and unless otherwise specified below, the terms "[DNS Query]"
+follow the specification for DNS queries as specified in
+[DNS Query and Response Defaults]. The handling of the DNS responses on the
+DNS queries follow, unless otherwise specified below, what is specified for
+[DNS Response] in the same specification.
+
+1. If the *Child Zone* is the root zone (".") then:
+   1. Output *[B01_CHILD_FOUND]* with zone name (".").
+   2. Exit the test case.
+
+2. Create [DNS queries][DNS Query]:
+   1. Query type SOA and query name *Child Zone* ("SOA Child Query").
+   2. Query type DNAME and query name *Child Zone* ("DNAME Query").
+
+3. Create the following empty sets:
+   1. Parent name server IP, parent zone name and DNS response
+      ("Parent Name Server IP").
```

Comment:
I could rename it -- it is just a name -- but I do not feel that "Parent Name Server" is better. Do you have a suggestion that captures all fields?

Reply:
Yeah, it covers more than just the parent. It's all the ancestors. Maybe "Ancestor Name Servers"? But then it's also not just the servers - it contains the delegation chain too. I guess it would make sense to name it "Lineage". Or "Delegation Chain"? But it's not just a simple chain either, it's more like a chain link fence. Well, there's a bunch of suggestions anyway. I guess my favorite is "Lineage", though it sticks out a bit from how all the other sets are named. Maybe it's the new black?

Reply:
Maybe just "Parents"? Maybe "Parenthood"?

(At first I wrote another response here. But I was confused thinking of the wrong set, so I deleted it.)

Reply:
Or is it "Nearest Authorities" or something?

Reply:
"Parent Information"? -- I test that.